contrib/utxo-tools/utxo_to_sqlite.py

```diff
@@ -16,6 +16,11 @@
 import sys
 import time
 
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '../../test/functional'))
+
+from test_framework.compressor import decompress_amount  # noqa: E402
+from test_framework.messages import deser_compact_size, deser_varint  # noqa: E402
+
 
 UTXO_DUMP_MAGIC = b'utxo\xff'
 UTXO_DUMP_VERSION = 2
```

Comment:
Could we import the `MAGIC_BYTES` from `test_framework.messages` as well (we might have to extend it with Testnet4)?